plugins/infinispan80/pom.xml

```diff
@@ -116,7 +116,7 @@
             </property>
          </activation>
          <properties>
-            <version.infinispan>8.0.1.Final</version.infinispan>
+            <version.infinispan>8.1.0.Beta1</version.infinispan>
```

Comment:
I didn't noticed it, will switch to ISPN 8.1 plugin once merged. Can someone review and merge it? Thanks!